angular_components/lib/model/menu/selectable_menu.dart

```diff
@@ -100,11 +97,22 @@ class SelectableMenuItem<ItemType> extends PropertyChangeNotifier
   @override
   final String secondaryLabel;
 
+  /// The constructor for a selectable [MenuItem].
+  ///
+  /// Selectable menu items can be managed with a [SelectionModel] from within
+  /// [MenuItemGroupWithSelection]. Selected menu items are marked in the UI and
+  /// can change the selection via user interactions.
+  ///
+  /// [itemSuffixes] - the list of suffixes rendered after the item content.
+  ///
+  /// [itemSuffix] - singular item suffix to be rendered after the item content;
+  ///     convenient way to pass a single item suffix in rather than
+  ///     constructing an ObservableList and using [itemSuffixes]. If
+  ///     [itemSuffixes] is also passed in, [itemSuffixes] takes precedence.
```

Comment:
I think that lines 109-111 might be over-indented, which will format the lines in code font (as part of a code block). Since this isn't a bulleted list item, the lines should be indented by at most two spaces.